addons/l10n_ro_edi/data/cius_ro_templates.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="cius_ro_InvoiceType" inherit_id="account_edi_ubl_cii.ubl_21_InvoiceType" primary="True">
+        <xpath expr="//*[local-name()='DocumentCurrencyCode']" position="after">
+            <cbc:TaxCurrencyCode
+                    xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2"
+                    xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+                    t-out="vals.get('tax_currency_code')"/>
```

Comment:
Should be added in ubl_cii version 2.0 because part of the standard template:
http://www.datypic.com/sc/ubl20/t-ns19_InvoiceType.html